### ISSUE TITLE
Doesn't work on node versions > 0.10

### ISSUE THREAD

**Issue by kpgarrod** [dharmafly/noodle#120]
When I try to run this library with nodejs > 0.10 I get an error: 'Error: Module did not self-register.'. It seems to come from xml2json which is an old version. Any chance of upgrading it please?

**Comment by premasagar** [comment]
Yes, that is right. That needs updating and testing. In the meantime, you can just remove xml2json from package.json before you run `npm install`, if you don't need XML parsing.
